README.md

```diff
@@ -307,6 +308,7 @@ demonstrates the setting and clearing of environment variables:
         ["MYENV1", "foo"],        # set MYENV1 to foo
         ["MYENV2", "foo", "bar"], # set MYENV2 to foo,bar
         ["MYENV3"],               # clear MYENV3
+        ["MYENV4", "!exec", "hostname", "-s"], # set MYENV4 to the output of `hostname -s`
```

Comment:
Could you please change this to work more like:
```
["MYENV4", "!hostname", "-s"], # set MYENV4 to the output of `hostname -s`
```
I think that this would be more inline with how other parts of homemaker works, and it would at the same time get rid of the need for `exec` keyword.